happy-travel/edo
# Markup according to the country where the hotel is located

Issue: https://github.com/happy-travel/agent-app-project/issues/1374
Insomnia: Edo -> Admin -> Markups -> Locations Markups
Tested through unit tests

Test case:
1) Add country destination markup through POST endpoint ../admin/location-markups and check validators
2) Start availability search for hotel from country which markup was added before
3) Check if markup was applied

JSON example:
{
"description": "test",
"functionType": 1,
"value": 20,
"currency": "USD",
"destinationScopeId": "RU",
"destinationScopeType": 2
}
